lib/qubicInterface.ts

```diff
@@ -93,6 +149,38 @@ export class QubicInterface {
     };
   }
 
+  async getTransactionWithPayload(
+    sourceSeed: string,
+    destinationPublicId: string,
+    amount: number,
+    tick: number,
+    inputType: number,
+    payload: Uint8Array,
+    payloadPackageSize: number,
+    asBase64: boolean = true
+  ) {
+    const sourceInfo = await this.qubicHelper.createIdPackage(sourceSeed);
+
+    const transaction = new QubicTransaction()
+      .setDestinationPublicKey(destinationPublicId)
+      .setSourcePublicKey(sourceInfo.publicId)
+      .setAmount(amount)
+      .setInputType(inputType)
+      .setTick(tick)
+      .setPayload({
+        getPackageData: () => payload,
+        getPackageSize: () => payloadPackageSize,
+      } as IQubicBuildPackage);
+
+    const tx = await transaction.build(sourceSeed);
+
```

Comment:
@Qubic-Hub according to what I see, build method returns already the trx package data.
I see tx is defined but not used and getPackageData is called below instead of using directly tx